static/js/dataPlaygroundList.js

```diff
@@ -0,0 +1,97 @@
+// Fetch the index.json file from the static folder and create a lunr index.
+let pagesIndex, searchIndex;
```

Comment:
This is the only code that includes a small update. The search index is now initialized here so that the select will work when a user chooses a license value.